includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -199,17 +182,114 @@ public function getData() {
 					]
 				]
 			],
+			'ddb_stores' => [
+				'image' => [
+					'type' => 'wds-svg',
+					'name' => 'wds-company-store-logo-ddb',
+					'caption' => [
+						'type' => 'text',
+						'value' => 'D&D Beyond'
+					],
+				],
+				'links' => [
+					[
+						'type' => 'link-image',
+						// 'image' is deprecated, use 'image-data' instead
```

Comment:
does anyone still use it?

Reply:
I'm not really sure, so I'll leave it for now. We'll throw it anyway sometime soon.

Reply:
soon ™️ :))